docs/source/about.rst

```diff
@@ -88,7 +88,7 @@ of PyMC. PyMC 2.2 was released in April 2012. It contained numerous bugfixes and
 optimizations, as well as a few new features, including improved output
 plotting, csv table output, improved imputation syntax, and posterior
 predictive check plots. PyMC 2.3 was released on October 31, 2013. It included
-Python 3 compatibility, improved summary plots, and some important bug fixes.
```

Comment:
This shouldn't be modified, as it's just the history of PyMC

Reply:
Ah, will revert!